.github/workflows/bump-deps.yaml

```diff
@@ -0,0 +1,69 @@
+# Description: Run this by visiting https://github.com/brave/brave-core/actions/workflows/bump-deps.yaml
+# and clicking the `Run workflow` button on the top-right.
+# You can also run it from cli by: `gh workflow run bump-deps.yaml -f ref=commit-hash`
+name: Bump DEPS
+on:
+  workflow_dispatch:
+    inputs:
+      dep:
+        description: "Dependency"
+        required: true
+        type: string
+        default: "vendor/web-discovery-project"
+      ref:
+        description: "New ref for dependency"
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  auto-bump:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: "0"
+
+      - name: auto-bump
+        run: |
+          set -x
+
+          # git config user.email "${{ github.actor }}@users.noreply.github.com"
+          # git config user.name "${{ github.actor }}"
+          # git config push.default simple
+          gh api /repos/brave/web-discovery-project/commits/main -q .sha
+
+          SED=sed
+          DATE=date
+
+          # Inputs
+          DEP="${{ inputs.dep }}"
+          DEPREF="${{ inputs.ref }}"
+          [[ "${DEP:?}" =~ ^[a-zA-Z/-]+$ ]] || { echo Input dependency validation failed; exit 1; }
+          [[ "${DEPREF:?}" =~ ^[a-z0-9]+$ ]] || { echo Dependency reference validation failed; exit 1; }
+
+          readarray -t milestones < <(gh api /repos/${GITHUB_REPOSITORY:?}/milestones -q ".[] | .title" | grep -Fv Closed)
+          branches=( "${milestones[@]/*Nightly/master}" )
+          branches=( "${branches[@]/ -*/}" )
+          echo "Branches to operate on: ${branches[*]}"
+          for branch in "${branches[@]}"
```

Comment:
if this GH action is intended to be used for other dependencies, then this is incorrect. Bumping the dep in beta/release should be optional.

is it possible to make a choice like this?
```
nightly
nightly+beta
nightly+beta+release
```